### PR TITLE
feat: enable feature level version negation and latest flag

### PR DESCRIPTION
**Pull request by psmul**
Closes: #5327

### CODE REVIEW

File: projects/core/src/features-config/utils/feature-config-utils.ts [hunk 1]
```diff
@@ -5,6 +5,15 @@ function isFeatureConfig(config: any): config is FeaturesConfig {
 }
 
 function isInLevel(level, version) {
+  if (level === '*') {
+    return true;
+  }
+  if (
+    version.indexOf('!') !== -1 &&
+    version.substr(1, version.length) === level
```
Comment (Platonn) [inline projects/core/src/features-config/utils/feature-config-utils.ts]: @dunqan What was the intention behind the `!` mark? I recall it was `lower than ...` (i.e. `!1.2` should mean `all versions lower than 1.2`). 
But I need to admit that for me it's not super-intuitive that `!` means `lower than`. ;)

Comment (dunqan) [reply projects/core/src/features-config/utils/feature-config-utils.ts]: 1. This logic should be moved to `isFeatureLevel` and `isFeatureEnabled` IMO.

3. `!` means negation, so not `lower than'` but like `notFeatureLevel` and `IsFeatureDisabled`.

File: projects/core/src/features-config/utils/feature-config-utils.ts [hunk 2]
```diff
@@ -21,7 +24,9 @@ function isInLevel(level, version) {
 
 export function isFeatureLevel(config: unknown, level: string): boolean {
   if (isFeatureConfig(config)) {
-    return isInLevel(config.features.level, level);
+    return level.indexOf('!') !== -1
```
Comment (dunqan) [inline projects/core/src/features-config/utils/feature-config-utils.ts]: We should check for ! at the beginning, so either ` return level.indexOf('!') === 0` or even simpler `level[0] === '!'`

File: projects/core/src/features-config/utils/feature-config-utils.ts [hunk 3]
```diff
@@ -30,6 +35,8 @@ export function isFeatureEnabled(config: unknown, feature: string): boolean {
     const featureConfig = config.features[feature];
```
Comment (dunqan) [inline projects/core/src/features-config/utils/feature-config-utils.ts]: `fatrure` here can now contain '!' at the beginning, so you should take it into an account

**Pull request merged**